produce_snp_matrix.py

```diff
@@ -1,46 +1,86 @@
 from glob import glob
-import subprocess
+import json
 import argparse
 import utils
 import os
 
-def snps(input_path, output_path):
+def download_test_case(case, download_path):
+    with open('./test.json') as f:
+        x =json.load(f)
+
+        try:
+            y = x[case]
+        except:
+            print(f"no case named {case}")
+        
+        num_samples = len(y)
+
+        while num_samples > 0:
```

Comment:
Could do `for sample in y.keys():` I think then you can get rid of lines 16, 19 and 34.  I would also consider changing the .json so that the instead of numbers for the highest level keys, you just have the sample name ...